src/parsers/transformer.rs

```diff
@@ -247,45 +248,44 @@ fn obfuscate_day(value: &str, table_name: &str) -> String {
     }
 }
 
-fn scramble(original_value: &str) -> String {
-    let mut chars = original_value.chars();
-    let mut output_buf = String::with_capacity(original_value.len());
-
-    let mut rng =
-        SmallRng::from_rng(rand::thread_rng()).unwrap_or_else(|_| SmallRng::from_entropy());
-
-    while let Some(current_char) = chars.next() {
-        if current_char == '\\' {
-            //The string contains a control character like \t \r \n
-            output_buf.push(current_char);
-            if let Some(c) = chars.next() {
-                output_buf.push(c);
+fn scramble(rng: &mut SmallRng, original_value: &str) -> String {
```

Comment:
I don't particularly love the code style of this new implementation, but whenever I tried to make it more elegant the compiler would end up generating slower code :|

Reply:
i think tbh, this is one of those times where performance is important as its called 10s of millions of times 🤷